test/e2e/tests/network/add-custom-network.spec.js

```diff
@@ -883,6 +903,10 @@ async function checkThatSafeChainsListValidationToggleIsOn(driver) {
   await driver.waitForSelector(accountOptionsMenuSelector);
   await driver.clickElement(accountOptionsMenuSelector);
 
+  if (process.env.METAMASK_BUILD_TYPE === 'mmi') {
```

Comment:
nice fix! I think that waiting for the menu to be fully rendered will fix the issue (in mmi we have this extra dashboard element).
However, at the moment we never enter inside this condition as this `process.env.METAMASK_BUILD_TYPE` returns `undefined`.
You can also check this locally when you run the test.


![Screenshot from 2024-07-15 12-11-21](https://github.com/user-attachments/assets/ae0c994a-8b3e-4c8a-bbf2-7059542b826f)

Also, if you look at the current ci logs, you will see that the below function is never executed.

How should we proceed from here? One easy thing could be to check for specific MMI element, such as the window title or the footer :thinking: 
Not sure if it's the best solution though :thinking: any other thoughts?

![Screenshot from 2024-07-15 12-22-08](https://github.com/user-attachments/assets/b6cef492-03e8-4e8c-bffa-618417c6e193)

Reply:
Thank you, @seaona for the comment. As you've already mentioned, the 'Portfolio Dashboard' menu appears dynamically only for the MMI build, which led me to consider a conditional check for the MMI build type. I also noticed that the conditional was skipped. Could you please share the syntax for checking the build type?

Also regarding your other point about the footer loading, it may not be effective due to the menu having a different loading time

Reply:
awesome!! :raised_hands: